src/frequenz/sdk/actor/data_sourcing/common.py

```diff
@@ -0,0 +1,66 @@
+"""Common types for the DataSourcingActor.
+
+Copyright
+Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+License
+MIT
+"""
+
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+from typing import Optional
+
+
+class ComponentMetric(Enum):
```

Comment:
This is first required by the `MicrogridApiSource`, so I would move it directly to `microgrid_source.py`.

Reply:
Closing so we have the discussion only in one place: https://github.com/frequenz-floss/frequenz-sdk-python/pull/52#discussion_r1005486650